cmd/transporter/javascript_builder.go

```diff
@@ -295,14 +301,35 @@ func (js *JavascriptBuilder) Build() error {
 
 // Run runs each of the transporter pipelines sequentially
 func (js *JavascriptBuilder) Run() error {
+	var g group.Group
 	for _, p := range js.pipelines {
-		err := p.Run()
-		if err != nil {
-			return err
-		}
+		g.Add(func() error {
+			return p.Run()
+		}, func(error) {
+			p.Stop()
```

Comment:
Just looked up the group package. Looks good enough for this use case. Any reason for using it instead of just a signal wait goroutine that stops the pipeline?

If we decide to keep the group package, we could log the error here so we have some user feedback when they SIGTERM.